mewarpx/mewarpx/utils_store/runsummary.py

```diff
@@ -0,0 +1,38 @@
+"""
+Utility class for printint out performance information after a run
+"""
+
+import time
+
+class RunSummary(object):
+    def __init__(self):
+        from pywarpx import callbacks
```

Comment:
Is there a reason you have to import `pywarpx` here rather than at the top of the file as usual?

Reply:
This one can be moved to the top of the file